pushflatpakscript/docker.d/init_worker.sh

```diff
@@ -12,6 +12,8 @@ test_var_set() {
 
 test_var_set 'FLATHUB_URL'
 
+export FLAT_MANAGER_CLIENT=/app/flat_manager_venv/bin/flat-manager-client
```

Comment:
Ah, this is neat, I should add a note to read this env var in `flathub.py` and read the executable from there rather than hardcoding it in the script. Now I understood what you said yesterday.

Later edit: or directly from `context.config['flat_manager_client']` as I saw you baked it there already.
Later edit: you've already done that, thanks!

Reply:
Hehehe. :)